[ADD] accounting: new shared accounts feature documentation (#11769)
odoo/documentation: content/applications/finance/accounting/get_started/consolidation.rst

:abbr:`CTA (Cumulative Translation Adjustment)`
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

In the case of multi-currency environments, CTA is used to use a single currency for all the
amounts in the report, the one from the **selected company**. CTA has three different methods
of calculation which are chosen following the **type of account** used in the transaction
which's currency has to be changed.

#. For **Equity** accounts **except the retained earnings**, the CTA is calculated with the historical
    rate which is the rate at the time of the transaction.
#. For **Income**, **Expense** and **retained earnings** accounts, the CTA is calculated with the
    average rate per day for the reported period.
#. For other accounts like **Assets** and **Liabilities** accounts, the CTA is calculated
    with the closing rate which is the last rate applied at the period's closing date.

The use of this system may result in non balanced reports. This is due to the fact that the CTA
can use a different rate for the payable and the receivable of a transaction which can lead to a
difference in the total amount of the report. This is normal and to be taken into account.

> .. note::
>     It is possible to disable CTA in the options of a report under the label :guilabel:`Currency
>     Translation`. If you don't want to use CTA, you can select the option :guilabel:`Use the most
>     recent rate at the date of the report` instead of :guilabel:`Use CTA`.

Reviewer: I wasn't able to find the "Currency translation" option, but I think adding some info (Go to Accounting --> ...) would be helpful.
```suggestion
.. note::
   CTA can be disabled in the report options under the :guilabel:`Currency Translation` label. To 
   avoid using CTA, select the option :guilabel:`Use the most recent rate at the date of the report` 
   instead of :guilabel:`Use CTA`.
```

   :alt: Confirmation wizard for the Unmerge Tool of the shared accounts feature.

A new account linked to each company will be created for the previously shared account.

.. _consolidation_import_account_mapping:

Import a Mapping
================

In order to **import an account mapping**, you can go to your chart of accounts in list view with
multiple companies selected.

First, export the fields you need using the :icon:`fa-cog` :guilabel:`Actions` button. For this
particular use case, you’ll need to add the **Code mapping/Code**, **Code Mapping/Company** and
**External ID fields**. No other field is required.

.. image:: consolidation/import_mapping_export.png
   :alt: Exporting the fields needed for the account mapping import.

Second, rework it briefly in a spreadsheet. For example, here, a few income accounts in "My Company
San Francisco" that will correspond to a single account in "My Belgian Company".

.. image:: consolidation/import_mapping_spreadsheet_editing.png
   :alt: Spreadsheet view of the exported xlsx file.

Third, reimport this file (xlsx or csv format) into Odoo via the :guilabel:`Import` button.

> .. image:: consolidation/import_mapping_import.png
>    :alt: Importing an xlsx or csv file for the accounts mapping.

Reviewer: This image is also pretty wide. The text is too small to read on lower-resolution screens.
```suggestion
```

> Finally, jump on any of the accounts you just reimported, and observe the codes now taking into
> account your mapping company per company.

Reviewer: ```suggestion
Finally, the codes now take into account the mapping company per company.
```

company** currency for all the amounts in the report. CTA has three different calculation methods,
chosen according to the **type of account** used in the transaction whose currency has to be
changed.

#. For **Equity** accounts, **except for retained earnings**, the CTA is calculated using the
   historical rate, which is the rate at the time of the transaction.
#. For **Income**, **Expense**, and **retained earnings** accounts, the CTA is calculated using the
   average daily rate for the reported period.
#. For other accounts, like **Assets** and **Liabilities** accounts, the CTA is calculated using
   the closing rate, which is the last rate applied at the period’s closing date.

This system may lead to unbalanced reports because the CTA can apply different rates for a
transaction's payable and receivable, causing a difference in the total amount. This is normal and
must be taken into account.

.. note::
   CTA can be disabled in the reports options under the :guilabel:`Currency Translation` label. Go
   to the Accounting App. In the Configuration drop-down select :guilabel:`Accounting Reports` and

>    chose the report for which you wish to disable the CTA and head in the :guilabel:`Options` page

Reviewer 2: ```suggestion
   chose the report for which you wish to disable the CTA, and head to the :guilabel:`Options` page
```

>    of the reprt's configuration. There, you can switch the :guilabel:`Currency Translation` option

Reviewer 2: ```suggestion
   of the report's configuration. There, you can switch the :guilabel:`Currency Translation` option
```
